python/django_bolt/testing/dbshare.py

```
@@ -0,0 +1,334 @@
+    # -- lifecycle ---------------------------------------------------------
+    def install(self) -> bool:
+        """Share the connections that hold an open transaction.
+
+        Returns True when sharing was installed. Nothing is shared when no
+        connection is in a transaction, because then the rows of the test are
+        committed already and the handler threads can read them by themselves.
+
+        Raises `SharedTestConnectionError` when another thread already shares
+        its connections. See the note on `_active_owner`.
+        """
+        global _active_owner, _active_depth
```

Ruff (0.16.1)

[warning] Using the global statement to update `_active_owner` is discouraged

(PLW0603)